### PR TITLE
add cloud provider user instructions

### PR DESCRIPTION
**Pull request by swoodward90**
## Checklist

- [ ] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [ ] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

File: pages/getting-started/cloud-shell-quickstart.md [hunk 1]
```diff
@@ -24,11 +24,76 @@ Navigate to [app.plural.sh/shell](https://app.plural.sh/shell) to access the Clo
 
 ### Set up a Cloud Provider
 
-Choose the option to set up your Cloud Provider in the browser. Plural can be set up on AWS, GCP, or Azure.
+Choose the option to continue your setup in the browser. If you'd prefer to use the CLI, choose "Local terminal" and switch to the CLI quickstart.
```
Comment (nickcernera) [inline pages/getting-started/cloud-shell-quickstart.md:27]: Probably "Choose an option"?

File: pages/getting-started/cloud-shell-quickstart.md [hunk 2]
```diff
@@ -24,11 +24,76 @@ Navigate to [app.plural.sh/shell](https://app.plural.sh/shell) to access the Clo
...
+
+{% tabs %}
+{% tab title="AWS" %}
+For AWS, you'll need to create or use a user with AdministratorAccess and create an access key for Plural to use.
+
+To create a new user, navigate to the IAM section of your AWS Console.
+
+![](</assets/cloud-shell-quickstart/iam-aws.png>)
+
+Select the `Users` tab and `Add users`. You should see the following screen:
```
Comment (nickcernera) [inline pages/getting-started/cloud-shell-quickstart.md:45]: I think the word "click" might be missing before "Add users"?

**Review by nickcernera — CHANGES_REQUESTED** [review]

File: pages/getting-started/cloud-shell-quickstart.md (hunk 1 above)
Comment (swoodward90) [reply pages/getting-started/cloud-shell-quickstart.md:27]: i'm assuming they're trying to set up in-browser, which is why it's directive

File: pages/getting-started/cloud-shell-quickstart.md (hunk 2 above)
Comment (swoodward90) [reply pages/getting-started/cloud-shell-quickstart.md]: addressed

**Review by nickcernera — APPROVED** [review]

**Review by avaidyanatha — COMMENTED** [review]
lgtm, trying out azure stuff now.

**Pull request merged**